src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -891,31 +841,23 @@ public void RemoveOrganelle(Hex hex)
     {
         int q = hex.Q;
         int r = hex.R;
-
         switch (Symmetry)
         {
             case MicrobeSymmetry.None:
-            {
                 RemoveOrganelleAt(new Hex(q, r));
                 break;
-            }
 
             case MicrobeSymmetry.XAxisSymmetry:
-            {
                 RemoveOrganelleAt(new Hex(q, r));
-
```

Comment:
Please keep the blank lines in this section. I much prefer blank lines before ifs unless the line before them computes a variables for use in the if.